CONFIGURATION.md

```diff
@@ -33,6 +33,7 @@ This server can be configured using the `workspace/didChangeConfiguration` metho
 | `pylsp.plugins.jedi_completion.resolve_at_most` | `integer` | How many labels and snippets (at most) should be resolved? | `25` |
 | `pylsp.plugins.jedi_completion.cache_for` | `array` of `string` items | Modules for which labels and snippets should be cached. | `["pandas", "numpy", "tensorflow", "matplotlib"]` |
 | `pylsp.plugins.jedi_definition.enabled` | `boolean` | Enable or disable the plugin. | `true` |
+| `pylsp.plugins.type_definition.enabled` | `boolean` | Enable or disable the plugin. | `true` |
```

Comment:
Should it be `jedi_` prefixed for consistency?

```suggestion
| `pylsp.plugins.jedi_type_definition.enabled` | `boolean` | Enable or disable the plugin. | `true` |
```

Reply:
Yes I think so, will do